docs/worker-node/install-singularity.md

```diff
@@ -140,6 +140,24 @@ unprivileged.
     such as Docker.  Disabling network namespaces may break other
     software, or limit its capabilities (such as requiring the
     `--net=host` option in Docker).
+    
+    Disabling network namespaces blocks the systemd PrivateNetwork
+    feature, which no default enabled EL 7 services depend on but some
+    EL 8 services do.  To check them all, look for PrivateNetwork in
```

Comment:
It took me a couple of tries to parse the second clause of this sentence.  Maybe just simplify it by saying that it's a feature more commonly seen across EL 8 services?

Reply:
I actually see it in several systemd files on EL 7 but they're all disabled services.  I'll spell that out in two sentences.